### DISCUSSION TITLE
changing database password

### DISCUSSION THREAD

**Discussion by salsero72** [symfony/symfony#50374] [Q&A] [answered]
Hello
sorry to bother you all with this really basic question but I'm really in a hurry.
I recently have been given the management of a doctrine application and I have to urgently change the database password 
I modified the existing 
`app/config/parameters.yml`
but when I change mysql password, the software keeps using the old one (app not working)
Reverting to the old password, everything goes back fine.

Is there anything to do after changing parameters?
A configuration flush or anything else?
Thanks

Here console about. I know version is old but I got hands on this today.
-------------------- -------------------------------------------
  Symfony
 -------------------- -------------------------------------------
  Version              3.4.2

**Comment by ghost** [comment]
From your issue description I understand that you don't use env variable for your password but you have it hardcoded inside parameters.yaml. I also suspect that you are talking about the production environment. In this case you have to rebuild the container (probably with `bin/console cache:clear --env prod`)

**Comment by salsero72** [comment]
Hello
thanks.
Are you talking about the commend described here?
https://symfony.com/doc/3.3/console/usage.html

php bin/console cache:clear --env=prod

Is there a way/command to revert to the previous environment configuration?
I'm a bit afraid that command refreshes settings I do not know where they
are taken from, that will break production site.

Thanks



Il giorno sab 20 mag 2023 alle ore 20:35 javaDeveloperKid <
***@***.***> ha scritto:

**Reply by ghost** [reply, marked as answer]
The cache is a thing that is autogenerated. Nobody should modify this by hand. So every cache after rebuild is based on the current configuration e.g. in yaml files. 
If you are very scared you can backup the cache directory and replace it if something goes wrong. You can also copy the project root somewhere else, execute `cache:clear` and check it every thing works. If yes then execute cache:clear inside the primary project root.

**Comment by salsero72** [comment]
A server reeboot can refresh settings from parameters.yml like 

php bin/console cache:clear --env=prod
?

**Reply by ghost** [reply]
No, server reboot will not refresh your app settings. Parameters are inlined when container is dumped (it's a physical PHP file).

**Comment by salsero72** [comment]
Thanks and sorry for my late reply.
Actually I didn't try yet since I am trying to clone the app to another hosting.
On the new hosting the following command is working 

php bin/console cache:clear --env=prod

Thanks
